_tools/lint_plugin.ts

```diff
@@ -38,5 +43,67 @@ export default {
         };
       },
     },
+    // https://docs.deno.com/runtime/contributing/style_guide/#error-messages
+    "error-message": {
+      create(context) {
+        return {
+          NewExpression(node) {
+            if (node.callee.type !== "Identifier") return;
+            const name = node.callee.name;
+            if (!name.endsWith("Error")) return;
+            const argument = node.arguments[0];
+            if (argument?.type !== "Literal") return;
+            const value = argument.value;
+            if (typeof value !== "string") return;
+
+            if (value.match(LEADING_LOWERCASE_REGEXP)) {
+              context.report({
+                node: argument,
+                message: "Error message starts with a lowercase.",
+                hint:
+                  "Capitalize the error message. See https://docs.deno.com/runtime/contributing/style_guide/#error-messages for more details.",
+                fix(fixer) {
+                  const newValue = argument.raw.at(0) +
+                    value[0].toUpperCase() +
+                    value.slice(1) +
+                    argument.raw.at(-1);
+                  return fixer.replaceText(argument, newValue);
+                },
+              });
+            }
+            if (value.match(TRAILING_PERIOD_REGEXP)) {
+              context.report({
+                node: argument,
+                message: "Error message ends with a period.",
+                hint:
+                  "Remove the period at the end of the error message. See https://docs.deno.com/runtime/contributing/style_guide/#error-messages for more details.",
+                fix(fixer) {
+                  const newValue = argument.raw.at(0) +
+                    value.slice(0, -1) +
+                    argument.raw.at(-1);
+                  return fixer.replaceText(argument, newValue);
+                },
+              });
+            }
+            if (value.match(PERIOD_REGEXP)) {
```

Comment:
```suggestion
            if (value.includes(".")) {
```

Reply:
I wonder if there's any case where `.` character is used for other purpose e.g. `obj.prop`, `1.5`, etc

Reply:
I implemented it as `value.includes(". ")` with a whitespace.